src/wix/test/WixToolsetTest.Converters/ConditionFixture.cs

```diff
@@ -102,6 +142,53 @@ public void FixPublishCondition()
             var actualLines = UnformattedDocumentLines(document);
             WixAssert.CompareLineByLine(expected, actualLines);
         }
+        [Fact]
+        public void FixPublishConditionWithComment()
+        {
+            var parse = String.Join(Environment.NewLine,
+                "<?xml version=\"1.0\" encoding=\"utf-16\"?>",
+                "<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi'>",
+                "  <Fragment>",
+                "    <UI>",
+                "      <Dialog Id='Dlg1'>",
+                "        <Control Id='Control1'>",
+                "          <Publish Value='abc'><!-- Comment 1 -->1&lt;2</Publish>",
+                "          <Publish Value='gone'><!-- Comment 2 -->1</Publish>",
+                "        </Control>",
+                "      </Dialog>",
+                "    </UI>",
+                "  </Fragment>",
+                "</Wix>");
+
+            var expected = new[]
+            {
+                "<Wix xmlns=\"http://wixtoolset.org/schemas/v4/wxs\">",
+                "  <Fragment>",
+                "    <UI>",
+                "      <Dialog Id=\"Dlg1\">",
+                "        <Control Id=\"Control1\">",
+                "          <Publish Value=\"abc\" Condition=\"1&lt;2\" />",
+                "          <!-- Comment 1 -->",
```

Comment:
Is it possible to put the comment before the element instead of after?

Reply:
Done.